test/commands/user/reputation_token/award_for_pull_requests_for_user_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class User::ReputationToken::AwardForPullRequestsForUserTest < ActiveSupport::TestCase
```

Comment:
@iHiD These tests are also quite important as they are testing the code that will be called each time a user's github handle is assigned or changes and on every pull request webhook event.